templates/taxi/car_detail.html

```diff
@@ -2,6 +2,12 @@
 
 {% block content %}
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
+   <td>
+     <a  class="btn btn-primary" href="{% url 'taxi:car-update' pk=car.id%}">Update</a>
+   </td>
+  <td>
+    <a class="btn btn-primary" href="{% url 'taxi:car-delete' pk=car.id%}">Delete</a>
+  </td>
```

Comment:
The `<td> `tag defines a standard data cell in an HTML table. I think it is a bad idea to use `<td>` in here.